react-frontend/src/pages/Signup/signup.tsx

```diff
@@ -105,7 +120,7 @@ export default function Signup() {
                         {/* Oauth2 Login */}
                         <div className="md:w-1/6 mt-6 md:mt-0">
                             <p className="text-sl mb-4 text-customYellow text-shadow-custom font-instrumentSans text-center">
-                                or<br/>continue with
+                                or<br />continue with
```

Comment:
little space here too

Reply:
It's because it open and close the context, it's two different things.